desc/objectives/_coils.py

```diff
@@ -1762,6 +1763,194 @@ def compute(self, params_1, params_2=None, constants=None):
         return Psi
 
 
+class LinkingCurrent(_Objective):
+    """Target the self-consistent poloidal linking current between the plasma and coils.
+
+    Assumes the coil topology does not change (ie the linking number with the plasma
+    is fixed).
+
+    Parameters
+    ----------
+    eq : Equilibrium
+        Equilibrium that will be optimized to satisfy the Objective.
+    coil : CoilSet
+        Coil(s) that are to be optimized.
+    grid : Grid, optional
+        Collocation grid containing the nodes to evaluate plasma current at.
+        Defaults to ``LinearGrid(M=eq.M_grid, N=eq.N_grid, NFP=eq.NFP, sym=eq.sym)``.
```

Comment:
no option for the coil grid discretization for the linking number calculation? Is that calculation pretty robust?

Reply:
Yeah since it's an integer pretty much any reasonable grid will do fine